bundle/Resources/config/fieldtypes.yml

```diff
@@ -163,3 +163,8 @@ services:
             - { name: ez.fieldFormMapper.value, fieldType: ezemail }
         calls:
             - [setFormType, ['Symfony\Component\Form\Extension\Core\Type\EmailType']]
+
+    EzSystems\RepositoryForms\FieldType\Mapper\ImageAssetFormMapper:
+        tags:
+            - { name: ez.fieldFormMapper.definition, fieldType: ezimageasset }
```

Comment:
`ez.fieldFormMapper.definition` is not needed as the class is not implementing `FieldDefinitionFormMapperInterface`